.github/copilot-instructions.md

```
@@ -2,520 +1,80 @@
+- [DSC Community Style Guidelines](instructions/dsc-community-style-guidelines.instructions.md) (`**`)
+  - Project-level guidelines: build scripts, testing, localization requirements
+  - Use for: General project structure and workflow guidance
 
-Desired State Configuration (DSC) class-based resource should always have
-its separate script file and the resource class name as the file name with
-the .ps1 extension, these files shall always be placed in the folder
-source/Classes.
+### PowerShell Code Guidelines
 
-### Parent classes
+- [DSC Community Style Guidelines - PowerShell](instructions/dsc-community-style-guidelines-powershell.instructions.md) (`**/*.psm1,**/*.psd1,**/*.ps1`)
```

markdownlint-cli2 (0.17.2)

Line length
Expected: 80; Actual: 152

(MD013, line-length)